.github/workflows/notified-of-spack-packages-update.yml

```diff
@@ -0,0 +1,66 @@
+name: Notification of spack_packages update
+on:
+  workflow_dispatch:
+    inputs:
+      spack-packages-version:
+        type: string
+        required: true
+        default: main
+        description: "Either a git tag or branch name for the ACCESS-NRI/spack_packages repository, which defaults to the main branch"
+      packages:
+        type: string
+        required: true
+        description: "packages from spack_packages that needs to be rebuilt"
+
+jobs:
+  update-spack-base-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name : Update Spack Base Image
+        uses: ./.github/workflows/build-and-push-image-base-spack.yml@main
+        with: 
+          spack-packages-version: ${{ github.event.inputs.spack-packages-version }}
+
+  generate-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      packages: ${{ steps.generate.outputs.packages }}
+      compilers: ${{ steps.generate.outputs.compilers }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with: 
+          python-version: '3.10'
+      - name: Generate model matrix
+        id: generate
+        run: python ./util/matrix-generator.py ${{ github.event.inputs.packages }}
```

Comment:
In order to verify this python script, should we have a separate test for it?

Reply:
Yes, we should. I'll add one.